test/Kestrel.Transport.FunctionalTests/Http2/H2SpecTests.cs

```diff
@@ -55,7 +55,7 @@ public static TheoryData<H2SpecTestCase> H2SpecTestCases
             get
             {
                 var dataset = new TheoryData<H2SpecTestCase>();
-                var toSkip = new[] { "http2/5.1/8" };
+                var toSkip = new string[] { /*"http2/5.1/8"*/ };
```

Comment:
just remove the entire variable.

Reply:
I'd rather not, if we have to come back and skip these tests for other reasons later I don't want to re-invent this.

Reply:
That's what the commit history is for isn't it?

Reply:
Searching for something that was removed is much more difficult.

Reply:
I'm not a fan of dead code but fair enough. Not really too bad since this is only in tests.